src/v2/view-builder/internals/BaseForm.js

```diff
@@ -133,6 +153,8 @@ export default Form.extend({
         if(!obj?.class || obj.class === 'INFO') {
           // add message as plain text
           this.add(`<p>${obj.message}</p>`, '.ion-messages-container');
+        } else if (obj.i18n?.key === OV_UV_ENABLE_BIOMETRIC_FASTPASS) {
```

Comment:
Shall this go Identify View or Terminal View?

Reply:
@haishengwu-okta Originally i added in Terminal view, but after I rebased there is another commit that move the show messages logic in the BaseForm https://github.com/okta/okta-signin-widget/commit/63e3765952b8493b48d1386a37d3f02fc423f6c8 So i changed the way I implemented it and moved it to the BaseForm.
Shall I override showMessages in TerminalView? cc @pradeepdewda-okta

Reply:
Yeah, I think Pradeep would give guideline.
IMHO `BaseForm` shall not have logic for specific flow.

Reply:
Haisheng is right.. This should not be included in BaseForm.

This should go to the relevant view that renders this flow..

Reply:
basically you can override `showMessages` in your respective views to display custom terminal messages..